version/version.go

```diff
@@ -17,6 +17,10 @@ limitations under the License.
 package version
 
 const (
+	// BundleVersionAnnotation is the annotation key used in the Gateway API inference extension CRDs to specify
+	// the installed Gateway API inference extension version.
+	BundleVersionAnnotation = "inference.networking.k8s.io/bundle-version"
+
 	// BundleVersion is the value used for labeling the version of the gateway-api-inference-extension.
-	BundleVersion = "v0.4.0-dev"
+	BundleVersion = "v0.5.0"
```

Comment:
nit: technically release v0.5.0 is still not out. 
maybe we should put rc2 suffix just to be on the safe side?
```suggestion
	BundleVersion = "v0.5.0-rc.2"
```

Reply:
updated